packages/grafana-runtime/src/services/panelAttentionSrv.ts

```diff
@@ -0,0 +1,31 @@
+export type PanelWithAttention = number | null;
+
+export interface PanelAttentionSrv<T = any> {
```

Comment:
A generic is used here because I cannot access `VizPanel` in runtime. There are issues with setters and getters when using `unknown`, not to speak of the typecasting required in that case.